src/MLDataPattern.jl

```diff
@@ -4,7 +4,9 @@ using LearnBase
 using MLLabelUtils
 
 using LearnBase: ObsDimension
-import LearnBase: nobs, getobs, getobs!, gettarget, gettargets, targets, datasubset, default_obsdim
+import StatsBase: nobs
+import LearnBase: getobs, getobs!, gettarget, gettargets, targets, datasubset, default_obsdim, DataView,
```

Comment:
Are these unexported by LearnBase 0.4?

`DataView, AbstractObsView, AbstractBatchView, DataIterator, AbstractDataIterator, ObsIterator, BatchIterator`

Reply:
AFAIK yes
```julia
julia> using LearnBase
julia> DataView
ERROR: UndefVarError: DataView not defined

julia> AbstractObsView
ERROR: UndefVarError: AbstractObsView not defined

julia> AbstractBatchView
ERROR: UndefVarError: AbstractBatchView not defined

julia> DataIterator
ERROR: UndefVarError: DataIterator not defined

julia> AbstractDataIterator
ERROR: UndefVarError: AbstractDataIterator not defined

julia> ObsIterator
ERROR: UndefVarError: ObsIterator not defined

julia> BatchIterator
ERROR: UndefVarError: BatchIterator not defined

julia> LearnBase.BatchIterator
LearnBase.BatchIterator
```

Reply:
There was quite a lot of unexporting in https://github.com/JuliaML/LearnBase.jl/compare/v0.3.0...v0.4.0

Reply:
I can export these in https://github.com/JuliaML/LearnBase.jl/pull/44 if that is the preferred end result. Seems like a lot got lost in translation between 0.3 and 0.4.